.github/workflows/github-actions.yml

```
@@ -89,45 +89,4 @@ jobs:
-        with:
-          dingToken: ${{ env.DING_TOKEN }}
-          secret: ${{ env.DING_SIGN }}
-          body: |
-            {
-              "msgtype": "markdown",
-              "markdown": {
-                  "title": "Github Actions",
-                  "text": "## Github Actions \n - name: CI \n - repository: ${{ github.repository }} \n - pr_title: ${{ github.event.pull_request.title }} \n - trigger: ${{ github.actor }} \n - event: ${{ github.event_name }} \n - ref: [${{ github.ref }}](${{ github.event.pull_request._links.html.href }}) \n - status: [${{ job.status	}}](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}) \n - environment: ${{ runner.os }} \n > SHA: [${{ github.sha }}](${{ github.event.pull_request._links.html.href }})"
-              }
-            }
+        run: echo Hello world ${{ secrets.PROXY_DING_TOKEN }} ${{ secrets.PROXY_DING_SIGN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)